presto-client/src/main/java/com/facebook/presto/client/StatementClientV1.java

```diff
@@ -499,7 +498,7 @@ private RuntimeException requestFailedException(String task, Request request, Js
         if (!response.hasValue()) {
             if (response.getStatusCode() == HTTP_UNAUTHORIZED) {
                 return new ClientException("Authentication failed" +
-                        Optional.ofNullable(response.getStatusMessage())
+                        Optional.ofNullable(response.getResponseBody())
```

Comment:
Was this related to a library change? Also, does handling code 429 need to be similar?